org.eclipse.m2e.core.ui/src/org/eclipse/m2e/core/ui/internal/archetype/ArchetypePlugin.java

```diff
@@ -113,7 +114,6 @@ protected void configure() {
     final ContainerConfiguration cc = new DefaultContainerConfiguration() //
         .setClassWorld(new ClassWorld("plexus.core", ArchetypeArtifactManager.class.getClassLoader())) //$NON-NLS-1$
         .setClassPathScanning(PlexusConstants.SCANNING_INDEX) //
-        .setAutoWiring(true) //
         .setName("plexus"); //$NON-NLS-1$
     container = new DefaultPlexusContainer(cc, logginModule);
     archetypeArtifactManager = container.lookup(ArchetypeArtifactManager.class);
```

Comment:
When trying this change locally this failed.
I assume this is a consequence of https://github.com/apache/maven-archetype/pull/206?
@laeubi any hint how to fix it?

Reply:
I'm not complete sure *what* failed, but disable autowiring does not looks like what one wants. so why should it be removed?

Reply:
The lookup fails with no such element. 
And auto-wiring is implied with our config.

Reply:
> Looking at the implementation autowiring is off by default and even if not I think we should make it explciti.

It's implied by the classPath-scanning mode:
https://github.com/eclipse-sisu/sisu-project/blob/8ea80a1289fe8efa8d8a95e3991c5bbf2c8f4bd6/org.eclipse.sisu.plexus/src/main/java/org/codehaus/plexus/DefaultContainerConfiguration.java#L126-L134

But we can also keep it explicit

> Was is maybe missing here is to enable .setJSR250Lifecycle(true)

At least locally, unfortunately this didn't help.

Reply:
Then "something" seems missing on the classpath for this to work :-\